backend/pkg/dbschemas/sql/mysql/queries/system.sql

```diff
@@ -214,6 +215,7 @@ SELECT
    c.TABLE_NAME AS table_name,
    c.COLUMN_NAME AS column_name,
    c.COLUMN_TYPE AS data_type,
+   c.COLUMN_TYPE AS column_type, -- same as data_type but includes things like the length, or set/enum information
```

Comment:
@alishakawaguchi - This column_type properly tracks mysql `set` and `enum` values. So I'm guessing that the mysql schema drift changes won't track if this changes. I created a ticket for this: NEOS-1791